themes/ddbasic/sass/configuration/_extend.scss

```diff
@@ -0,0 +1,24 @@
+
```

Comment:
Describe component contents/purpose, why are these styles grouped? What is the common denominator?